hawc/apps/study/views.py

```diff
@@ -116,6 +118,43 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class StudyClone(BaseUpdate):
+    """
+    Copy studies along with all study information from one assessment to another.
+    """
+
+    template_name = "study/study_clone.html"
+    # parent_template_name = "assessment"
+    # parent_model = Assessment
+    model = Assessment
+    form_class = forms.StudyCloneForm
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        context["study_data"] = {}
+        assessments = Assessment.objects.all().user_can_view(
```

Comment:
This list will be massive in many cases, suggest making it smaller by adding a src assessment and only showing references from the src assessment.